include/fiction/utils/layout_utils.hpp

```diff
@@ -408,42 +415,71 @@ CoordinateType random_coordinate(CoordinateType coordinate1, CoordinateType coor
  * @param cell_se The southeast SiQAD cell defining the ending point of the area.
```

Comment:
There are still some SiQAD namings in the comments here. Maybe do a quick CTRL + SHIFT + F for "siqad" across the project to see what else there is to replace.